src/time_bucket.c

```diff
@@ -283,7 +284,7 @@ ts_time_bucket_by_type(int64 interval, int64 timestamp, Oid timestamp_type)
 			bucket_function = ts_date_bucket;
 			break;
 		default:
-			elog(ERROR, "invalid time_bucket Oid %d", timestamp_type);
+			elog(ERROR, "invalid time_bucket type \"%s\"", format_type_be(timestamp_type));
```

Comment:
This might not make sense because if the type doesn't exist, `format_type_be` will generate a generic cache lookup failure/error on its own. Is this what we want?

Reply:
i think the intent here is for it to error when being called with types that we don't support in time_bucket like float or decimal not that the type in general is invalid, but i dont think you can reach this part without a bug somewhere in our code